packages/distance/index.d.ts

```diff
@@ -0,0 +1,113 @@
+/**
+ * Transpose a note by an interval. The note can be a pitch class.
+ *
+ * This function can be partially applied.
+ *
+ * @param {String} note
+ * @param {String} interval
+ * @return {String} the transposed note
+ * @example
+ * import { tranpose } from "tonal-distance"
+ * transpose("d3", "3M") // => "F#3"
+ * // it works with pitch classes
+ * transpose("D", "3M") // => "F#"
+ * // can be partially applied
+ * ["C", "D", "E", "F", "G"].map(transpose("M3)) // => ["E", "F#", "G#", "A", "B"]
+ */
+export declare function transpose(note: string, interval: string): string | ((el:string)=>string);
```

Comment:
Here's where type aliases shines:

```ts
export declare function transpose(note: note, interval: interval): note | ((el:interval)=>note);
```

Anyway, the second parameter is optional:
```suggestion
export declare function transpose(note: string, interval?: string): string | ((el:string)=>string);
```